Document valid forms of repository argument

I'm trying to set up GitHub Actions for CI on Git LFS.  We want to clone Git itself so that we can test against older versions, but it appears that I'm invoking things incorrectly.

I expected that the `repository` argument could be an arbitrary Git remote, and I've tried `https://github.com/git/git` and `https://github.com/git/git.git`, both of which are public, but neither appears to work.  The message I get is as follows:

```
  with:
    repository: https://github.com/git/git
    path: git
    ref: master
    clean: true
##[error]Specified argument was out of the range of valid values.
Parameter name: https://github.com/git/git
##[error]Exit code 1 returned from process: file name '/home/runner/runners/2.157.0/bin/Runner.PluginHost', arguments 'action "GitHub.Runner.Plugins.Repository.CheckoutTask, Runner.Plugins"'.
```

If you're interested in an example run, the "Build with latest Git" target on https://github.com/git-lfs/git-lfs/runs/198689891 should be illustrative.

Could you document the valid forms of the repository argument so that folks can figure out how to use this functionality?

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@ Refer [here](https://github.com/actions/checkout/blob/v1/README.md) for previous
 ```yaml
 - uses: actions/checkout@v2-beta
   with:
-    # Repository name
+    # Repository name with owner. For example, actions/checkout
     # Default: ${{ github.repository }}
     repository: ''
 

--- a/action.yml
+++ b/action.yml
@@ -2,7 +2,7 @@ name: 'Checkout'
 description: 'Checkout a Git repository'
 inputs: 
   repository:
-    description: 'Repository name'
+    description: 'Repository name with owner. For example, actions/checkout'
     default: ${{ github.repository }}
   ref:
     description: >

